connexion/security/security_handler_factory.py

```diff
@@ -351,43 +351,48 @@ def wrapper(request, token, required_scopes):
             if need_to_add_context:
                 kwargs[self.pass_context_arg_name] = request.context
```

Comment:
In the verify_security function, we are no longer using `request.context` and use `request.scope['context']` - why this is not the case here?

Reply:
I think it should be updated here as well indeed.